packages/web/app/src/components/ui/callout.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { cva, type VariantProps } from 'class-variance-authority';
+import { cn } from '@/lib/utils';
+import { InfoCircledIcon } from '@radix-ui/react-icons';
+
+const TypeToEmoji = {
+  default: '💡',
+  error: '🚫',
+  info: <InfoCircledIcon className="h-6 w-auto" />,
+  warning: '⚠️',
+};
```

Comment:
if we can try to utilize icons from all of these from radix that would be better because on some browsers I have seen emoji's don't render properly

Reply:
Done! 🥇